src/wazuh_modules/inventory_harvester/src/wcsModel/wcsClasses/agent.hpp

```diff
@@ -12,6 +12,7 @@
 #ifndef _AGENT_WCS_MODEL_HPP
 #define _AGENT_WCS_MODEL_HPP
 
+#include "host.hpp"
```

Comment:
Do we need this change?

Reply:
We don't, I forgot to remove it. Thank you.